moped-database/migrations/1649874244000_alter_project_list_view/up.sql

```diff
@@ -0,0 +1,46 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+ SELECT mp.project_uuid,
+    mp.project_id,
+    mp.project_name,
+    mp.project_description,
+    mp.project_description_public,
+    mp.ecapris_subproject_id,
+    mp.project_importance,
+    mp.project_order,
+    mp.current_status,
+    mp.timeline_id,
+    mp.current_phase,
+    mp.end_date,
+    mp.project_length,
+    mp.fiscal_year,
+    mp.capitally_funded,
+    mp.project_priority,
+    mp.date_added,
+    mp.added_by,
+    mp.is_retired,
+    mp.milestone_id,
+    mp.task_order,
+    COALESCE(mp.status_id, 0) AS status_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ','::text) AS project_team_members,
+    mp.updated_at,
+    me.entity_name AS project_sponsor,
+    string_agg(DISTINCT me2.entity_name, ', '::text) AS project_partner,
+        CASE
+            WHEN ((mp.status_id = 0) OR (mp.status_id IS NULL)) THEN NULL::text
```

Comment:
`NULL::text` seems like it should just be `NULL`, similar to above.

Reply:
I didn't change this one because the other ones were strings being cast into text, but in this case its NULL being cast into text. does this return `"null"` vs `null`?

Reply:
I waffled on mentioning this one too. I should have waffled on the side of not saying anything 🧇 . 

```
select
    null as plain_null,
    null::text as text_casted_null,
    pg_typeof(null) as type_of_plain_null,
    pg_typeof(null::text) as type_of_text_casted_null;
```
Yields:

![image](https://user-images.githubusercontent.com/3653206/164088066-f7d01948-a94e-4eba-9612-14796f328917.png)

Thank you for catching this and asking I dig deeper. 🙏 👍

Reply:
no, thank you!